### ISSUE TITLE
Space Suit will not retain any stored oxygen given

### ISSUE THREAD

**Issue by Odogo** [MrScautHD/Beyond-Earth#226]
### Mod loader

Forge

### Minecraft version

1.19.2

### Beyond Earth version

6.3i

### Modloader version

Forge: 1.19.2-43.2.4

### The latest.log file

https://cdn.odogo.lol/uploads/2023/02/latest.log

### Issue description

After successfully crafting a Space Suit, I stuck it into an Oxygen Loader to have it "use" all of the oxygen it was created over time, and give me nothing in return. In the video [https://cdn.odogo.lol/uploads/2023/02/javaw_YYapAgnATA.mp4], you can see exactly that. The oxygen would get consumed into the suit, but the suit would _store_ any of it.

Attempting to see if I can use the prefilled creative suit, which also just resets back to 0 as well.

### Steps to reproduce

1. Host a server with the Beyond Earth mod with the specific version details.
2. Grab a _unfilled_ space suit
3. Attempt to store oxygen in it, by means granted by the mod itself or with Mekanism or other fluid management mods.

OTHERWISE

2. Grab a filled space suit, and watch that precious oxygen dissipate immediately.

### Other information

The "lastest.log" is from the server itself. A mod list can be provided if necessary.

**Comment by rdavis93** [comment]
Having the same issue here. Came to report

**Comment by st0x0ef** [comment]
Does it work with the 6.4 version?

**Comment by rdavis93** [comment]
> 

I am using 1.19.2-6.4 and having the same issues

**Comment by Odogo** [comment]
> Does it work with the 6.4 version?

No, the 6.4 version does not fix the issue. It repeats the same issue stated before.

**Comment by st0x0ef** [comment]
Ok...
